Essentials/src/main/java/com/earth2me/essentials/commands/Commandhome.java

```diff
@@ -81,7 +81,7 @@ public void run(final Server server, final User user, final String commandLabel,
                     } else {
                         showError(user.getBase(), new Exception(tl("noHomeSetPlayer")), commandLabel);
                     }
-                } else if (homes.isEmpty()) {
+                } else if (homes.isEmpty() || finalPlayer.getHome(homes.get(0)) == null) {
```

Comment:
@JRoy if the first home is from a deleted world, players can't see their homes on /homes because of this condition

Reply:
Thanks for spotting this, I'll get this fixed sooner than later.